web/package.json

```diff
@@ -42,7 +42,7 @@
   },
   "scripts": {
     "start": "cross-env PORT=7001 craco start",
-    "build": "craco build",
+    "build": "NODE_ENV=production craco build",
```

Comment:
1. Why add this? (debug v.s. prod?)
2. Does this work for Windows too?

Reply:
`craco` uses `NODE_ENV` to recognize environments, but it is in production mode when NODE_ENV is not set,  so I think we can remove it.

Reply:
It's redundant.